queue_services/entity-filer/tests/unit/test_worker/test_correction_special_resolution.py

```diff
@@ -83,6 +88,29 @@ def __init__(self, file_content):
         'familyName': 'Doe',
         'additionalName': ''
     }
+    correction_data['filing']['correction']['parties'] = [
+        {
+            'officer': {
+                'partyType': 'person',
+                'firstName': 'hello',
+                'lastName': '1234'
+            },
+            'roles': [
+                {
+                    'roleType': 'Director',
+                    'appointmentDate': '2023-07-05'
+                }
+            ],
+            'mailingAddress': {
+                'streetAddress': '4343 55 Avenue Cres',
+                'addressCity': 'Innisfail',
+                'addressRegion': 'AB',
+                'postalCode': 'T4G 1X4',
+                'addressCountry': 'CA'
+            }
+        }
+    ]
```

Comment:
when you have the code working to add a completing party and tie it to he SR correction filing, can you check that completing party gets populated properly?

Reply:
yes, added:
```
existing_filing_party_roles = PartyRole.get_party_roles_by_filing(correction_filing_id, end_date_time.date())
assert any(party_role.role == PartyRole.RoleTypes.COMPLETING_PARTY.value
                   for party_role in existing_filing_party_roles)
```